lib/ipmi_main.c

```diff
@@ -371,6 +373,17 @@ ipmi_main(int argc, char ** argv,
 	while ((argflag = getopt(argc, (char **)argv, OPTION_STRING)) != -1)
 	{
 		switch (argflag) {
+		case 'i':
+			if (bind_intf) {
+				free(bind_intf);
+				bind_intf = NULL;
```

Comment:
there is `free_n()` function for that in `include/ipmitool/helper.h`, please use it